kbs/src/prometheus_exporter.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2025 Red Hat
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use lazy_static::lazy_static;
+use prometheus::{core::Collector, TextEncoder};
+use std::sync::Mutex;
+
+lazy_static! {
+    pub static ref instance: Mutex<PrometheusExporter> = Mutex::new(PrometheusExporter::new());
```

Comment:
Let's use [tokio::sync::RwLock](https://docs.rs/tokio/latest/tokio/sync/struct.RwLock.html) rather than `std::sync::Mutex`. 

It can improve performance and efficiency in two aspects: allowing requests at the same time and asynchronous IO.